nuxeo-coldstorage-web/i18n/messages.json

```diff
@@ -15,5 +15,11 @@
   "moveDocumentsContentsToColdStorage.confirm": "Confirm",
   "moveDocumentsContentsToColdStorage.error": "Error moving the documents contents to cold storage.",
   "moveDocumentsContentsToColdStorage.success": "Documents contents moved to cold storage successfully.",
-  "moveDocumentsContentsToColdStorage.tooltip": "Send the selected items to cold storage"
-}
+ "moveDocumentsContentsToColdStorage.tooltip": "Send the selected items to cold storage",
+
+  "deleteContentFromColdStorage.description" : "This will delete definitely the document content from cold storage.",
```

Comment:
`definitely` is not proper English for the meaning you want, please have a native English speaker review the messages.